contract/models/contract.py

```diff
@@ -633,17 +664,24 @@ def _get_recurring_create_func(self, create_type="invoice"):
             return self.__class__._recurring_create_invoice
 
     @api.model
-    def _cron_recurring_create(self, date_ref=False, create_type="invoice"):
+    def _cron_recurring_create(
+        self, date_ref=False, create_type="invoice", domain=None
```

Comment:
Changing the number of arguments is conflicting, as any override of this method won't know about the new argument, and it will fail, or at least, ignore this new argument. Can you avoid it?